include/exadg/solvers_and_preconditioners/multigrid/multigrid_preconditioner_base.cpp

```diff
@@ -471,8 +471,7 @@ MultigridPreconditionerBase<dim, Number, MultigridNumber>::
       dealii::Functions::ZeroFunction<dim, MultigridNumber> zero_function(
         dof_handler->get_fe().n_components());
 
-      auto const & mapping_dummy =
-        dof_handler->get_fe().reference_cell().template get_default_linear_mapping<dim>();
```

Comment:
I found this syntax often on deal.II, but I do not know it. My compiler also got confused. Why not take the right one?

Reply:
Internally, the function does the same. Both won't work for mixed meshes. So we will need to change this line at most then.